gspread/worksheet.py

```diff
@@ -2402,6 +2407,7 @@ def _duplicate(
         client: HTTPClient,
         spreadsheet_id: str,
         sheet_id: int,
+        spreadsheet: Any,
```

Comment:
why no typing?

Reply:
because we won't keep it in my mind (I won't let it go sorry :stuck_out_tongue: ) 
and typing it requires importing `Spreadsheet`, which already imports `Worksheet` which creates a circular dependency.

We can work around that using some conditional imports, I'll try it

Reply:
I managed to type it correctly :upside_down_face: